### ISSUE TITLE
About the Code

### ISSUE THREAD

**Issue by myt889** [NUST-Machine-Intelligence-Laboratory/CoLDL#1]
Hi,  thank you for your work. I have some question about the code. In coldl.py,  what the  "t" represents? I can't find the it in your dataloader files.

             if cfg.dataset == 'cifar100' and cfg.noise_type != 'clean':
                    t = torch.tensor(dataset['train'].noisy_labels)
                else:
                    t = torch.tensor(dataset['train'].targets)
                labels2learn1 = torch.full(size=(dataset['n_train_samples'], n_classes), fill_value=0.0)
                labels2learn1.scatter_(dim=1, index=torch.unsqueeze(t, dim=1), value=1.0 * 10)
                labels2learn2 = labels2learn1
